bin/dev.js

```diff
@@ -1,5 +1,3 @@
-#!/usr/bin/env node_modules/.bin/tsx
```

Comment:
Let's not remove this shebang please, as then it'd break running the CLI from the apify-cli folder

That said, it's sad we cannot make it work in all ways 😢

Reply:
What exactly do you mean by break running the CLI from the apify-cli folder? You should be able to run it from anywhere using the `./bin/dev.sh` file.

Reply:
Running the file directly instead of from the .sh file, but I suppose its fine either way. Could also make it an npm script (npm run dev) in the future